### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
# Description

Updated github logo to support both light and dark backgrounds

__Related to:__ `add dropdown to readme`

__Any expected test failures?__

---

Add a `[X]` to relevant checklist items

## ❔ This change

- [ ] adds a new feature
- [ ] fixes breaking code
- [X] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

- [ ] Refactored code ([sourcery](https://sourcery.ai/))
- [ ] Tested code locally
- [ ] Precommit installed and run before pushing changes
- [ ] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [ ] Updated GitHub [README](../README.md)
- [ ] Updated the projects overview page on Notion

---

## 🧪 Test Configuration

- OS:
- Python version:
- Neptune version:
- Affected libraries with version:

### CODE REVIEW

**Review by kshitij12345 — COMMENTED** [review]
Moving the table under `detail` will affect search.

I tried to find `pytorch` with browser search on this new file but didn't get any hit.

image

**Comment by SiddhantSadangi** [comment]
> Moving the table under `detail` will affect search.
> 
> I tried to find `pytorch` with browser search on this new file but didn't get any hit.
> 
should have checked for this before 🤦‍♂️

will revert the dropdown changes

**Comment by SiddhantSadangi** [comment]
@normandy7 , @Patrycja-J - any other changes needed from a content/marketing perspective?

File: README.md [hunk 1]
```diff
@@ -163,18 +165,16 @@ You can run every example with zero setup as an "ANONYMOUS" Neptune user (no reg
 | Docker | [![docs]](https://docs.neptune.ai/usage/docker) | [![neptune]](https://app.neptune.ai/o/common/org/showroom/e/SHOW-3102/dashboard/Metrics-a7de0cf6-3b35-44b5-9d7a-791e5617a44e) | [![github]](how-to-guides/neptune-docker/scripts)
 | GitHub Actions | [![docs]](https://docs.neptune.ai/usage/github_actions) | | [![github]](https://github.com/neptune-ai/automation-pipelines)
 
-### Amazon SageMaker
+### Amazon Sagemaker
```
Comment (normandy7) [inline README.md:168]: The original spelling should be correct.

**Review by normandy7 — APPROVED** [review]

**Pull request merged**